dom.bs

```diff
@@ -5523,6 +5548,19 @@ or <var>A</var>'s <a for=tree>root</a> is a <a for=/>shadow root</a> whose
 <a for=ShadowRoot>mode</a> is "<code>open</code>" and <a for=DocumentFragment>host</a> is an
 <a>unclosed node</a> of <var>B</var>.
 
+<p>To <dfn export lt="retarget|retargeting">retarget</dfn> an object <var>A</var> against an object
```

Comment:
Now, I am using "an object A" here because it is consistent to other definitions around here.